sdk/tables/azure-tables/src/EdmInt64.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * A class that allows you to represent a 64-bit signed integer as a string.
+ */
+export class EdmInt64 {
+  private _value: string = "";
+
+  constructor(integer: string) {
+    this.value = integer;
+  }
+
+  /**
+   * Returns the value of the integer
+   */
+  get value(): string {
+    return this._value;
+  }
+
+  set value(integer: string) {
+    this._value = integer;
```

Comment:
We might consider doing some form of validation here to enforce the string actually represents a number and that number is inside the range of an Int64?

Reply:
Maybe we can use [Number.isInteger()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isInteger) ?

Reply:
Right now it only checks if the number is an integer, but not if it's in the range of an int64.
Do you guys know if there's a way to check if a number is within the range of a 64 bit integer in JavaScript without the use of BigInts or 3rd party libraries?

Reply:
I don't think there's a great way to detect if a string number is too big without some third party lib (or doing something ourselves.)